app/AvailableLayer.php

```diff
@@ -113,4 +113,8 @@ public static function createFromArray($props) {
     public function entity_type() {
         return $this->belongsTo('App\EntityType');
     }
+
+    public function user_layer_option() {
```

Comment:
@v1r0x is this correct in this branch? It looks like it's not used and meant for something else.

Reply:
Yes and no :D
It is all needed for the next release of the map plugin. That's why I added it to this PR instead of adding a new one for this small feature :man_shrugging: